scripts/testpypi-smoke.py

```
@@ -0,0 +1,83 @@
+
+def main(argv: list[str]) -> int:
+    if len(argv) != 2:
+        print("usage: testpypi-smoke.py <version>", file=sys.stderr)  # noqa: T201
+        return 2
+    version = argv[1]
+
+    # (1) Install PLAIN cement from TestPyPI. The zero-dependency core resolves
+    # from TestPyPI alone; do NOT install the [cli] extra (its jinja2/pyyaml
+    # deps live only on real PyPI and would force an extra real-PyPI index,
+    # widening the dependency-confusion surface — Pitfall 2 / T-05.4-01).
+    subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)